src/ast.rs

```diff
@@ -93,16 +93,6 @@ impl PartialEq for ExprKind {
             _ => false,
         }
     }
-
-    fn ne(&self, other: &Self) -> bool {
-        match (self, other) {
-            (Self::Atom(l0), Self::Atom(r0)) => l0 != r0,
-            (Self::List(l0), Self::List(r0)) => l0 != r0,
-            (Self::Func(l0), Self::Func(r0)) => *l0 as usize != *r0 as usize,
-            (Self::Lambda(l0), Self::Lambda(r0)) => l0 != r0,
-            _ => false,
-        }
-    }
```

Comment:
Why was this removed?

Reply:
nvm, found it - https://rust-lang.github.io/rust-clippy/master/index.html#partialeq_ne_impl